src/CBT.After.Any.Targets/CBT.After.Any.Targets.nuproj

```diff
@@ -15,10 +15,7 @@
   </ItemGroup>
   <ItemGroup>
    <Dependency Include="CBT.Microsoft.Common.targets">
-      <Version>1.0.0</Version>
```

Comment:
Would we want newer versions of the CBT.After.Any.Targets module to still extend the CBT.Traversal module?  Since we're working to deprecate the module, it seems like a good idea to do this but I'm not sure if it will cause people to have to move off of it sooner than they want.